src/config/dark-sites.config

```diff
@@ -502,6 +502,10 @@ humblegames.com
 hyper.is
 hyperbeam.dev
 i3wm.org
+iaas.store
+*.iaas.store
+iaasdev.ru
```

Comment:
This page ends with some page placeholder which is white as hell when I tried it.
You sure it's correct address?

Reply:
im sorry for your eyes :) , we have added your extension to the placeholder (via cdn). the address is correct and all pages on it will support a dark theme

Reply:
ok... now placeholder is dark too (hope the final page will be dark too)